platforms/enterprise/enterprise/src/main/java/org/gradle/internal/enterprise/impl/GradleEnterpriseAutoAppliedPluginRegistry.java

```diff
@@ -70,7 +70,24 @@ private PluginRequestInternal createGradleEnterprisePluginRequest() {
             getScriptDisplayName(),
             artifact,
             null,
-            AUTO_APPLIED
+            AUTO_APPLIED,
+            develocityPluginRequest()
+        );
+    }
+
+    private static PluginRequestInternal develocityPluginRequest() {
+        ModuleIdentifier moduleIdentifier = DefaultModuleIdentifier.newId(AutoAppliedGradleEnterprisePlugin.GROUP, AutoAppliedGradleEnterprisePlugin.DEVELOCITY_PLUGIN_ARTIFACT_NAME);
+        ModuleVersionSelector artifact = DefaultModuleVersionSelector.newSelector(moduleIdentifier, AutoAppliedGradleEnterprisePlugin.VERSION);
+        return new DefaultPluginRequest(
+            AutoAppliedGradleEnterprisePlugin.DEVELOCITY_PLUGIN_ID,
+            AutoAppliedGradleEnterprisePlugin.VERSION,
+            false,
+            null,
+            getScriptDisplayName(),
+            artifact,
+            null,
+            AUTO_APPLIED,
+            null
```

Comment:
❓ Do we need to duplicate the whole plugin request here? I think it would make more sense to have a smaller structure with only the necessary coordinates/plugin id.

Reply:
Fair. Extract a `PluginCoordinates` class in https://github.com/gradle/gradle/pull/27365/commits/0aa9ab06572a0826005a7ec8a18e8d0cb12b1e1e